indico/modules/events/timetable/legacy.py

```diff
@@ -157,6 +158,7 @@ def serialize_break_entry(self, entry, management=False):
                      'sessionId': block.session_id if block else None,
                      'sessionCode': block.session.code if block else None,
                      'sessionSlotId': block.id if block else None,
+                     'sessionBlockEntryId': entry.parent.id if entry.parent else None,
```

Comment:
Maybe, just for the sake of not having more concepts in the legacy serializer, we could call this `sessionSlotEntryId`.